app/MMB/src/menu/MMBDeeplink.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
```

Comment:
I think we should try to avoid prefixing all components with MMB

Reply:
@tbergq Improved. Also please check it more heavily since my webview is not working properly again (problem with my machine probably)...